flowtorch/parameters/__init__.py

```diff
@@ -7,7 +7,8 @@
 """
 
 from flowtorch.parameters.base import Parameters
+from flowtorch.parameters.coupling import DenseCoupling
 from flowtorch.parameters.dense_autoregressive import DenseAutoregressive
 from flowtorch.parameters.tensor import Tensor
 
-__all__ = ["Parameters", "DenseAutoregressive", "Tensor"]
+__all__ = ["Parameters", "DenseAutoregressive", "Tensor", "DenseCoupling"]
```

Comment:
By the way, you can do the autogen imports script for adding a new Parameters class too :)